README.md

```diff
@@ -12,3 +12,118 @@ libraryDependencies += "com.gu" %% "content-api-firehose-client" % "x.y"
+parameter to [this] (https://github.com/guardian/crier/blob/master/cloudformation.json) cloudformation file as so:
+
+```
+"TEAM_NAMEAccountNumber": {
+    "Description": "Account number for the TEAM_NAME team",
+    "Type": "String"
+}
+```
+
+And then add a corresponding assumeRole command to the [cross account role] (https://github.com/guardian/crier/blob/master/cloudformation.json#L100)
+as such: 
+
+```
+{
+    "Action": "sts:AssumeRole",
+    "Effect": "Allow",
+    "Principal": {
+        "AWS": {
+            "Fn::Join": [ "", [ "arn:aws:iam::", { "Ref": "TEAM_NAMEAccountNumber" }, ":root" ]]
+        }
+    }
+}
+```
+
+Once this been done, someone on the team will review, merge and update for you to begin. Create an IAM policy in your 
+project's cloudformation file as so: 
+
+ 
+ - STAGE - e.g PROD or CODE
+ - APP_NAME - e.g. my-application
+ - MODE - e.g. live or preview
+ - CAPI_ACCOUNT_NUMBER - The AWS account number for CAPI. Ask someone from the CAPI team to provide you with this.
+ - STREAM_NAME - The AWS Kinesis stream name of the CAPI events stream. Ask someone from the CAPI team to provide you with this.
+
+```json
+  "CrierDynamoDBPolicy": {
+    "Type": "AWS::IAM::Policy",
+    "Properties": {
+      "PolicyName": "CrierDynamo",
+      "PolicyDocument": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "dynamodb:*"
+            ],
+            "Resource": [
+              {
+                "Fn::Join": [
+                  "",
+                  [
+                    "arn:aws:dynamodb:*:CAPI_ACCOUNT_NUMBER:table/content-api-firehose-v2-STAGE_APP_NAME-MODE-",
+                    {
+                      "Ref": "Stage"
+                    },
+                    "*"
+                  ]
+                ]
+              }
+            ]
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "kinesis:Get*",
+              "kinesis:List*",
+              "kinesis:Describe*"
+            ],
+            "Resource": [
+              "arn:aws:kinesis:*:CAPI_ACCOUNT_NUMBER:stream/STREAM_NAME"
+            ]
+          }
+        ]
+      },
+      "Roles": [
+        {
+          "Ref": "RootRole"
+        }
+      ]
+    }
+  }
+```
+
+Once this has been done you will be able to create a `ContentApiFirehoseConsumer` in your application as such:
+
+```scala
+
+  val credsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("capi"),
+    new InstanceProfileCredentialsProvider()
+  )
+
+val contentApiFirehoseConsumer: ContentApiFirehoseConsumer = new ContentApiFirehoseConsumer(
+    app = "my-application",
+    mode: "live",
+    suffix = None,
+    streamName = "GET_STREAM_NAME_FROM_SOMEONE_IN_CAPI",
+    stage = "PROD",
+    kinesisCredentialsProvider = credsProvider,
+    dynamoCredentialsProvider = credsProvider,
```

Comment:
Here we want them to use separate creds providers for Kinesis and Dynamo.

* The one for Kinesis should be an `STSAssumeRoleSessionCredentialsProvider` (or a chain that includes one)
* The one for Dynamo should look like the `credsProvider` you give as an example above (although the profile name would not be "capi")